data/Ashish-simpleCoder.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Ashish Prajapati",
+  "bio": "Web Developer in JavaScript | ReactJS | Typescript-React",
+  tags: ["Typescript", "Javascript", "ReactJS", "Open Source"]
```

the following corrections are needed for the JSON to pass:
```suggestion
  "tags": ["Typescript", "Javascript", "ReactJS", "Open Source"],
```